Services/Simulation/JavascriptInterpreter.cs

```diff
@@ -59,22 +55,7 @@ public class JavascriptInterpreter : IJavascriptInterpreter
         /// Modifies the internal device state with the latest values.
         /// </summary>
         public void Invoke(
```

Comment:
I would write the code differently, having one method invoking a script from storage and one method invoking a script from disk. For instance, I would check if the script is on disk or storage before calling the interpreter (although I'd need to try how the code looks like, doing it here might be ok too).

If possible, I'd looks for a way to avoid Task.Wait and Task.Result, and do the usual await/async